lib/ui/hash_codes.dart

```diff
@@ -2,122 +2,33 @@
-/// ```
+@Deprecated(
+  'Use Object.hash() instead '
+  'This feature was deprecated in v2.9.0-0.1.pre'
+)
 int hashValues(
-  Object? arg01,            Object? arg02,          [ Object? arg03 = _hashEnd,
-  Object? arg04 = _hashEnd, Object? arg05 = _hashEnd, Object? arg06 = _hashEnd,
-  Object? arg07 = _hashEnd, Object? arg08 = _hashEnd, Object? arg09 = _hashEnd,
-  Object? arg10 = _hashEnd, Object? arg11 = _hashEnd, Object? arg12 = _hashEnd,
-  Object? arg13 = _hashEnd, Object? arg14 = _hashEnd, Object? arg15 = _hashEnd,
-  Object? arg16 = _hashEnd, Object? arg17 = _hashEnd, Object? arg18 = _hashEnd,
-  Object? arg19 = _hashEnd, Object? arg20 = _hashEnd ]) {
-  int result = 0;
-  result = _Jenkins.combine(result, arg01);
-  result = _Jenkins.combine(result, arg02);
-  if (!identical(arg03, _hashEnd)) {
-    result = _Jenkins.combine(result, arg03);
-    if (!identical(arg04, _hashEnd)) {
-      result = _Jenkins.combine(result, arg04);
-      if (!identical(arg05, _hashEnd)) {
-        result = _Jenkins.combine(result, arg05);
-        if (!identical(arg06, _hashEnd)) {
-          result = _Jenkins.combine(result, arg06);
-          if (!identical(arg07, _hashEnd)) {
-            result = _Jenkins.combine(result, arg07);
-            if (!identical(arg08, _hashEnd)) {
-              result = _Jenkins.combine(result, arg08);
-              if (!identical(arg09, _hashEnd)) {
-                result = _Jenkins.combine(result, arg09);
-                if (!identical(arg10, _hashEnd)) {
-                  result = _Jenkins.combine(result, arg10);
-                  if (!identical(arg11, _hashEnd)) {
-                    result = _Jenkins.combine(result, arg11);
-                    if (!identical(arg12, _hashEnd)) {
-                      result = _Jenkins.combine(result, arg12);
-                      if (!identical(arg13, _hashEnd)) {
-                        result = _Jenkins.combine(result, arg13);
-                        if (!identical(arg14, _hashEnd)) {
-                          result = _Jenkins.combine(result, arg14);
-                          if (!identical(arg15, _hashEnd)) {
-                            result = _Jenkins.combine(result, arg15);
-                            if (!identical(arg16, _hashEnd)) {
-                              result = _Jenkins.combine(result, arg16);
-                              if (!identical(arg17, _hashEnd)) {
-                                result = _Jenkins.combine(result, arg17);
-                                if (!identical(arg18, _hashEnd)) {
-                                  result = _Jenkins.combine(result, arg18);
-                                  if (!identical(arg19, _hashEnd)) {
-                                    result = _Jenkins.combine(result, arg19);
-                                    if (!identical(arg20, _hashEnd)) {
-                                      result = _Jenkins.combine(result, arg20);
-                                      // I can see my house from here!
```

Comment:
i'm sad that my joke is going away :-(

Reply:
Sorry to take that away 😢 . I loved it btw! Made me laugh IRL.

Reply:
Your joke lives on 🖖🏾